web/packages/teleport/src/Discover/Server/DiscoveryConfigSsm/DiscoveryConfigSsm.tsx

```diff
@@ -174,94 +196,111 @@ export function DiscoveryConfigSsm() {
-            mt={3}
-          >
-            Next
-          </ButtonSecondary>
-        )}
-        {scriptUrl && (
-          <ButtonSecondary onClick={() => setScriptUrl('')} mt={3}>
-            Edit
-          </ButtonSecondary>
-        )}
-      </StyledBox>
-      {showRestOfSteps && (
+    <Validation>
+      {({ validator }) => (
         <>
+          <Header>Setup Discovery Config for Teleport Discovery Service</Header>
+          {cfg.isCloud ? (
+            <Text>
+              The Teleport Discovery Service can connect to Amazon EC2 and
+              automatically discover and enroll EC2 instances.{' '}
+              <SsmInfoHeaderText />
+            </Text>
+          ) : (
+            <Text>
+              Discovery config defines the setup that enables Teleport to
+              automatically discover and register instances.{' '}
+              <SsmInfoHeaderText />
+            </Text>
+          )}
+          {cfg.isCloud && <SingleEc2InstanceInstallation />}
           <StyledBox mt={4}>
-            <H3>Step 2</H3>
-            <P>
-              Attach AWS managed{' '}
-              <ExternalLink
-                target="_blank"
-                href="https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonSSMManagedInstanceCore.html"
-              >
-                AmazonSSMManagedInstanceCore
-              </ExternalLink>{' '}
-              policy to EC2 instances IAM profile. The policy enables EC2
-              instances to use SSM core functionality.
-            </P>
-          </StyledBox>
-          <StyledBox mt={4}>
-            <H3>Step 3</H3>
-            <P>
-              Each EC2 instance requires{' '}
-              <ExternalLink
-                target="_blank"
-                href="https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent-status-and-restart.html"
-              >
-                SSM Agent
-              </ExternalLink>{' '}
-              to be running. The SSM{' '}
-              <ExternalLink
-                target="_blank"
-                href={`https://${selectedRegion}.console.aws.amazon.com/systems-manager/fleet-manager/managed-nodes?region=${selectedRegion}`}
+            <header>
+              <H3>Step 1</H3>
+              <Subtitle3>
+                Select the AWS Region that contains the EC2 instances that you
+                would like to enroll
+              </Subtitle3>
+            </header>
+            <Box mb={-5}>
+              <AwsRegionSelector
+                onFetch={(region: Regions) => setSelectedRegion(region)}
+                clear={clear}
+                disableSelector={!!scriptUrl}
+              />
+            </Box>
+            {selectedRegion && (
+              <Box mb={3} mt={5}>
+                <P>
+                  You can filter for EC2 instances by their tags. If no tags are
+                  added, Teleport will use a wildcard to enroll all EC2
+                  instances.
```

Comment:
There's a couple of non-obvious things about label matchers:
- there's the special `*: [*]`
- you can use the a wildcard for any tag key: `project: *`
- you can have multiple values for the same tag key: `project: a, b`

The 1st is abstracted by the UI, by explicitly saying that removing everything will match every resource.
I'm not sure we should be explicit about the underlying representation (`*`/ _wildcard_). I would just say this:

```suggestion
                  You can filter for EC2 instances by their tags. If no tags are
                  added, Teleport will enroll all EC2 instances.
```

For the 2nd and 3rd, explaining all of this in this screen might be too complex/boring.

Either we ignore those, or we might add a informational panel explaining them:
it stays out of the user's way, but if they are interested in advanced customization, they can read the details in the informational panel.

Reply:
Sounds good 👍 
Keeping it simple looks good to me.

Reply:
i think we agreed to change something here, but i'm not sure i'm seeing it 🤔

Reply:
Its there now @kimlisa !